src/extension/content-script/batteries/Reddit.ts

```diff
@@ -0,0 +1,49 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+const urlMatcher = /^https:\/\/www.reddit\.com\/user\/(\w+).*/;
+
+function battery(): void {
+  const descriptionElement = document.querySelector<HTMLMetaElement>(
+    'head > meta[name="description"]'
+  );
+  const imageUrl = document.querySelector<HTMLImageElement>(
+    `img[alt="User avatar"]`
```

Comment:
it seems this does not match on all pages. e.g. https://www.reddit.com/user/bitdex/
but maybe we do not need this actually. It seems there is also a meta tag for the image: 
```
document.querySelector('head > meta[property="og:image"]')
```

and I think this one should be already extracted through the `getOriginData()`. so maybe we we can even just rely on that. 
Can you test that? Or alternatively explicitly use the `og:image` here. (`document.querySelector('head > meta[property="og:image"]')?.content`)